airbyte-integrations/connectors/source-recurly/metadata.yaml

```diff
@@ -16,9 +16,16 @@ data:
   name: Recurly
   registries:
     cloud:
-      enabled: false
+      enabled: true
     oss:
-      enabled: false
+      enabled: true
+  releases:
+    breakingChanges:
+      1.0.0:
+        message:
+          Version 1.0.0 introduces a number of schema updates to the Recurly connector.
+          To ensure a smooth upgrade, please refresh your schemas and reset your data before resuming syncs.
+        upgradeDeadline: "2024-03-05"
```

Comment:
Put this date as a placeholder since March 5th was the stated deadline for using the sunsetting connectors. Let me know if this should be changed!